ui-tests/src/test/java/org/openmrs/reference/page/ManageLocationTagsPage.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * <p>
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.reference.page;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+import org.openqa.selenium.TimeoutException;
+
+public class ManageLocationTagsPage extends Page {
+   
+    protected static final String PAGE_URL = "adminui/metadata/locations/manageLocationTags.page";
+    private static final By ADD_NEW_LOCATION_TAG = By.cssSelector("input.button");
```

Comment:
Is the above a button?

Reply:
Have done the needful in the commit with id `4da39cc`